packages/datagateway-search/src/facet/components/facetPanel/toggleableFilterItem.component.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Box,
+  Checkbox,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from '@mui/material';
+import React from 'react';
+
+interface ToggleableFilterItemProps {
+  classificationLabel: string;
+  count: number;
+  selected: boolean;
+  onSelect: (classificationLabel: string, selected: boolean) => void;
+}
+
+function ToggleableFilterItem({
+  classificationLabel,
+  count,
+  selected,
+  onSelect,
+}: ToggleableFilterItemProps): JSX.Element {
+  return (
+    <ListItemButton
+      dense
+      disableGutters
+      sx={{ px: 1.5 }}
+      selected={selected}
+      aria-label={`${
+        selected ? 'Remove' : 'Add'
+      } ${classificationLabel} filter`}
+      aria-selected={selected}
+      onClick={() => onSelect(classificationLabel, !selected)}
+    >
+      <ListItemIcon sx={{ minWidth: 'auto' }}>
+        <Checkbox
+          size="small"
+          disableRipple
+          sx={{ padding: 0, paddingRight: 1 }}
+          checked={selected}
+          inputProps={{
+            'aria-label': `${
+              selected ? 'Remove' : 'Add'
+            } ${classificationLabel} filter`,
+          }}
+          onChange={(event) =>
+            onSelect(classificationLabel, event.target.checked)
+          }
```

Comment:
Remove this `onChange` handler in favour of the one on the list item - as otherwise clicking on the checkbox specifically will cause 2 chips to get added